.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+variables:
+  CURRENT_CI_IMAGE: v7679051-11c779a8
+  KUBERNETES_SERVICE_ACCOUNT_OVERWRITE: lemur
+
+stages:
+  - build-stage-image
+  - build-prod-image
+
+build-stage-image:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/images:$CURRENT_CI_IMAGE
+  stage:build-stage-image
+  when: on_success
+  # the 3h timeout is required for images compiling large projects for ARM using qemu
```

Comment:
Does this comment still apply to us? Not sure we are building an ARM compatible container image here.